Applications/SlicerApp/Testing/Python/DICOMReaders.py

```diff
@@ -97,23 +97,27 @@ def test_Part1DICOM(self):
         "fileName": "Mouse-MR-example-where-GDCM_fails.zip",
         "name": "Mouse-MR-example-where-GDCM_fails",
         "seriesUID": "1.3.6.1.4.1.9590.100.1.2.366426457713813178933224342280246227461",
-        "expectedFailures": ["GDCM", "Archetype"]
+        "expectedFailures": ["GDCM", "Archetype"],
+        "voxelValueQuantity": "(DCM, 110852, MR signal intensity)",
+        "voxelValueUnit": "(UCUM, [hnsf'U], Hounsfield unit)"
```

Comment:
@pieper @ihnorton, as native speakers, what do you think: Do both volumeNode.SetVoxelValueUnit(...) and volumeNode.SetVoxelValueUnits(...) sound good to you for setting voxel unit, such as HU or MBq/g, etc.?